tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "compileOnSave": true,
+  "compileOnSave": false,
```

Comment:
Good catch! Were there a bunch of .js files just getting written for no reason? lol